cirq-google/cirq_google/calibration/xeb_wrapper.py

```diff
@@ -38,7 +39,8 @@ def _maybe_multiprocessing_pool(
         yield None
         return
 
-    with multiprocessing.get_context('spawn').Pool(processes=n_processes) as pool:
+    ctx = multiprocessing.get_context('spawn' if os.name == 'nt' else 'fork')
```

Comment:
Does this fail by raising `ValueError` on the `get_context` call, as described in [the docs](https://docs.python.org/3/library/multiprocessing.html#multiprocessing.get_context)? If so, could perhaps use that instead of switching on the os name:
```python
try:
    ctx = multiprocessing.get_context('spawn')
except ValueError:
    ctx = multiprocessing.get_context('fork')
```
Of course, if it doesn't fail until later when trying to actually spawn something, then might need to switch on the os as you've done.